ketcher-autotests/tests/Templates/Template-Library/template-library.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { Page, test } from '@playwright/test';
+import {
+  getCoordinatesOfTheMiddleOfTheScreen,
+  pressButton,
+  resetCurrentTool,
+  selectTopPanelButton,
+  STRUCTURE_LIBRARY_BUTTON_NAME,
+  takeEditorScreenshot,
+  TopPanelButton,
+} from '@utils';
+
+async function setDisplayStereoFlagsSettingToOn(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Stereochemistry', { exact: true }).click();
+  await pressButton(page, 'IUPAC style');
+  await page.getByRole('option', { name: 'On' }).click();
+  await pressButton(page, 'Apply');
+}
+
+async function switchIgnoreChiralFlagSetting(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Stereochemistry', { exact: true }).click();
+  await page.getByText('Ignore the chiral flag').click();
+  await pressButton(page, 'Apply');
+}
+
+async function placePhenylalanineMustard(page: Page, x: number, y: number) {
+  await pressButton(page, STRUCTURE_LIBRARY_BUTTON_NAME);
+  const phenylalanineLocator = page.locator(
```

Comment:
I guess, you need to indicate, why have you chosen exactly that template

Reply:
Done